sema/rocreator/rocreator.yml

```
@@ -0,0 +1,53 @@
+    - roles: owner, contributor, contact, funder
+
+content:
+  ignorecase: yes | no | tolower
+  includes:
+    - "**/*.csv"
+    - "**/*tsv"
+    - "**/*.xml"
+  excludes:
+    - .*ignore
+    - .github/
+    - README.md 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)